src/demo/App.js

```diff
@@ -1,40 +1,107 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import TextareaAutosize from 'react-autosize-textarea';
 
 import ReactRRuleGenerator from '../lib';
 import './index.css';
 import githubLogo from './github_logo.png';
 
-const App = () => (
-  <div>
-    <div className="app-navbar">
-      <a href="https://github.com/fafruch/react-rrule-generator">
-        &lt; go back to
-        {' '}
-        <img className="app-navbar-ghlogo" src={githubLogo} alt="Github logo" />
-        /fafruch
-      </a>
-
-      <iframe
-        title="github-star"
-        src="https://ghbtns.com/github-btn.html?user=fafruch&repo=react-rrule-generator&type=star&count=true&size=medium"
-        frameBorder="0"
-        scrolling="0"
-        width="72px"
-        height="20px"
-      />
-    </div>
-    <div className="app-header">
-      <h1>React RRule Generator</h1>
-    </div>
-
-    <div className="app-desc">
-      Recurence rules generator form built with React
-    </div>
-
-    <div className="app">
-      <ReactRRuleGenerator />
-    </div>
-  </div>
-);
+class App extends Component {
+  state = {
+    rrule: '',
+    isCopied: false,
+  };
+
+  handleChange = (newRRule) => {
```

Comment:
This is correct and expected. With it you do not need `onChange={newRRule => this.handleChange(newRRule)}`. All you need is `onChange={this.handleChange}` Please check the template.